ArduCopter/mode_systemid.cpp

```diff
@@ -0,0 +1,319 @@
+#include "Copter.h"
+
+#if MODE_SYSTEMID_ENABLED == ENABLED
+
+/*
+ * Init and run calls for systemId, flight mode
+ */
+
+const AP_Param::GroupInfo ModeSystemId::var_info[] = {
+
+    // @Param: _AXIS
+    // @DisplayName: System identification axis
+    // @Description: Controls which axis are being excited
+    // @User: Standard
+    // @Values: 0:None, 1:Input Roll Angle, 2:Input Pitch Angle, 3:Input Yaw Angle, 4:Recovery Roll Angle, 5:Recovery Pitch Angle, 6:Recovery Yaw Angle, 7:Disturbance Roll, 8:Disturbance Pitch, 9:Disturbance Yaw, 10:Rate Roll, 11:Rate Pitch, 12:Rate Yaw, 13:Mixer Roll, 14:Mixer Pitch, 15:Mixer Yaw, 16:Mixer Thrust
```

Comment:
@lthall you indicate here that the user has 16 available input types, but the enumeration above only covers 13?